pkg/controller/job/job_controller.go

```diff
@@ -782,10 +783,14 @@ func (jm *Controller) syncJob(ctx context.Context, key string) (rErr error) {
 		return err
 	}
 
+	activePods := controller.FilterActivePods(logger, pods)
```

Comment:
I see no need to move this line, you can leave it when `jobCtx` is computed.

Reply:
you are correct.